### ISSUE TITLE
travis is broken due to line endings

### ISSUE THREAD

**Issue by lbillingham** [lbillingham/geomag_wdc_web_app_interface#6]
if I use a linux vm with `firefox` to get data from `[the data portal](http://wdc.bgs.ac.uk/dataportal/)

we get 
```shell
$ file ngk2015.wdc
ASCII text, with CRLF line terminators
```
i.e. windows line endings.

However, by using the REST API from `linux` we get `unix` line endings (`LF` line endings).
[n.b. check line endings using the API from windows, vbelieve get `CLRF` but have not checked explicitly] .

Is it appropriate to `dos2unix` the 'known good' data downloaded from via the web interface?

Who gets to run the functional tests and have them pass? Win users or unix users or how do we modify things appropriately?

**Comment by lbillingham** [comment]
Can think of 2 ways of dealing with this:

1. have 2 versions of _all_ 'known good' files
    * 1 with `LF` line  endings 
    * 1 with `CRLF` line endings
2. Read both files explicitly line-by-line and rely on imlicit conversion to `\n` line endings by `open`

c.f. [this SO answer](http://stackoverflow.com/a/23038606/3751373)
```python
def cmp_lines(path_1, path_2):
    l1 = l2 = ' '
    with open(path_1, 'r') as f1, open(path_2, 'r') as f2:
        while l1 != '' and l2 != '':
            l1 = f1.readline()
            l2 = f2.readline()
            if l1 != l2:
                return False
    return True
```

**Comment by lbillingham** [comment]
conclusion after talking to @petes is that looking after a 10 line function is easier than duplicate oracle files.

Also need to investigate the line endings coming out of the data portal.  it depends on which data type you ask for (wdc, json, iaga2002,...) -> issue in the BGS WDC JIRA

**Comment by lbillingham** [comment]
fixed by 7d7b57c4b96bd4a2536baea5820547626c2df506
